examples/express-bots/index.js

```diff
@@ -0,0 +1,42 @@
+import arcjet, { detectBot, shield } from "@arcjet/node";
+import express from "express";
+
+const app = express();
+const port = 3000;
+
+const aj = arcjet({
+  // Get your site key from https://app.arcjet.com and set it as an environment
+  // variable rather than hard coding.
+  key: process.env.ARCJET_KEY,
+  rules: [
+    // Protect against common attacks with Arcjet Shield
+    shield({
+      mode: "LIVE", // will block requests. Use "DRY_RUN" to log only
+    }),
+    // Detect bots with the ability to allow or deny subsets
+    detectBot({
+      mode: "LIVE",
+      deny: ["CURL"] // explicitly deny the curl command
+      // allow: [] // explicitly allow bots in the list while denying all others
+    }),
+  ],
+});
+
+app.get('/', async (req, res) => {
+  const decision = await aj.protect(req);
+
+  if (decision.isDenied() && decision.reason.isBot()) {
+    res.writeHead(400, { "Content-Type": "application/json" });
+    res.end(JSON.stringify({
+      error: "You are a bot",
+      detected: decision.reason.denied[0]
+    }));
+  } else {
```

Comment:
Since we're adding shield as a rule as well should we do something with a potential shield response in the example?

Reply:
I really don't want to. Adding branching for shield takes away from what the example is showing.

We add shield to every example because we originally "auto added" shield and so we keep it in every example to mirror that behavior.

Reply:
Would it be worth even adding a comment indicating that you should add handling for shield? My concern is that a user would just copy the example, assume that its set up to handle both rules since they're both in there, then be confused as to why shield does nothing

Reply:
These examples are not boilerplates to copy, they are just for testing specific functionality works and continues to work in the SDK during development.

Please add feedback to #72 which discusses how we need to create consolidated examples that show everything.

Reply:
We started these as examples to copy, but they became more part of our test suite. We still refer to them in some places as examples, but we're going to deprioritize that and focus on the docs code samples and other "full" example apps. Should probably rename them from examples as well, but this is for #72.